web/src/main/java/org/mskcc/oncotree/api/MainTypesApi.java

```diff
@@ -46,7 +46,7 @@
         meta.setCode(200);
         resp.setMeta(meta);
 
-        Version v = VersionUtil.getVersionOrRealtime(version);
+        Version v = (version == null) ? VersionUtil.getDefaultVersion() : VersionUtil.getVersion(version);
```

Comment:
Ok using the trinary operator (a ? b : c) .. but if the expression was any more complex I might prefer an if/then